src/input/mod.rs

```diff
@@ -1120,6 +1120,12 @@ impl State {
                         );
                         // Decide on action if first update
                         if first_update {
+                            let mut natural_scroll = true;
```

Comment:
So according to the libinput docs, natural_scroll should always be an option for touchpad and thus the scroll_config as well as the natural_scroll property should always be set here: https://github.com/pop-os/cosmic-comp/blob/ed64e26faf4b97221f0883bc1113f997acbadc50/src/config/input_config.rs#L56

**But** safe-guarding is certainly a good idea, though I don't think I agree with the default. libinput seems to generally set this to `false` if no device specific config overrides this (e.g. for Apple Touchpads). As such, I am inclined to say this should be `false` as well for a consistent experience.

```suggestion
                            let mut natural_scroll = false;
```